layouts/container_footer.html

```diff
@@ -2,6 +2,6 @@
 
 <footer>
   <p class="pull-left">
-    &copy; Prometheus Authors 2016
+    &copy; Prometheus Authors 2017
```

Comment:
good idea! I can make that change

out of curiosity, what templating language are we using?